### PR TITLE
docs(empty): add note about fantasy-land Monoid spec

### PR DESCRIPTION
**Pull request by char0n**

### CODE REVIEW

**Comment by CrossEye** [comment]
LGTM :herb:

Objections, anyone?

File: src/empty.js [hunk 1]
```diff
@@ -8,8 +8,9 @@ var _isString = require('./internal/_isString');
 /**
  * Returns the empty value of its argument's type. Ramda defines the empty
  * value of Array (`[]`), Object (`{}`), String (`''`), and Arguments. Other
- * types are supported if they define `<Type>.empty` and/or
- * `<Type>.prototype.empty`.
+ * types are supported if they define `<Type>.empty`,
+ * * `<Type>.prototype.empty` or implement the
```
Comment (kedashoe) [inline src/empty.js]: is there an extra `* ` at the beginning of this line?

Comment (char0n) [reply src/empty.js]: @kedashoe yep it was. Fixed in additional commit. Thanks.

**Comment by buzzdecafe** [comment]
:cow2:

**Pull request merged**